.github/workflows/static-analysis.yml

```diff
@@ -13,4 +13,4 @@ jobs:
       - name: Run linters
         uses: golangci/golangci-lint-action@v3.4.0
         with:
-          version: v1.47.2
+          version: v1.51.2
```

Comment:
you should also fix the .golangci.yml file to replace 1.18 with 1.19, and maybe fix a few warnings. I can have a look tomorrow if you prefer

Reply:
done